app/models/manageiq/providers/openshift/container_manager.rb

```diff
@@ -3,6 +3,13 @@
 class ManageIQ::Providers::Openshift::ContainerManager < ManageIQ::Providers::Kubernetes::ContainerManager
   DEFAULT_EXTERNAL_LOGGING_ROUTE_NAME = "logging-kibana-ops".freeze
 
+  has_one :infra_manager,
```

Comment:
Does it makes sense for this to just live in the base class, or is this intentionally overriding the class name?

Reply:
This is here because we have to override the class_name to be Openshift::InfraManager so the correct class gets created